src/core/ext/filters/max_age/max_age_filter.cc

```diff
@@ -85,26 +91,97 @@ struct channel_data {
   grpc_connectivity_state connectivity_state;
   /* Number of active calls */
   gpr_atm call_count;
+  /* 'idle_state' holds the states of max_idle_timer and channel idleness.
```

Comment:
Thanks for the detailed comment here. If you don't mind, it would help if you write a brief description of what each state means..
especially the `MAX_IDLE_STATE_SEEN_EXIT_IDLE` and `MAX_IDLE_STATE_SEEN_ENTER_IDLE` .

Reply:
I'll add the description in #14122